### PR TITLE
Input 

### PR DESCRIPTION
**Pull request by SergeyGhazaryan**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,24 +1,81 @@
...
-import './App.css';
+import React, { useState } from 'react';
+import InputField from '../src/Component/InputType';
+
+import './Style/styles.css';
 
 function App() {
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
+  const [age, setAge] = useState('');
```
Comment (Karen-Avdalyan) [inline src/App.js:9]: age is not string

File: src/App.js [hunk 2]
```diff
@@ -1,24 +1,81 @@
...
+  const [country, setCountry] = useState('');
+
+  const [newFirstName, setNewFirstName] = useState('');
+  const [newLastName, setNewLastName] = useState('');
```
Comment (Karen-Avdalyan) [inline src/App.js:13]: this is not acceptable way to save data

**Review by Karen-Avdalyan — CHANGES_REQUESTED** [review]

File: package.json [hunk 3]
```diff
@@ -34,5 +34,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "@types/react": "^17.0.2"
```
Comment (Karen-Avdalyan) [inline package.json:39]: Are you using here types? I don't think so.

File: src/App.js [hunk 4]
```diff
@@ -1,24 +1,88 @@
...
+      setVisible(true);
+      const payload = {
+        firstName: firstName,
+        lastName: lastName,
+        age: age,
+        country: country,
+      };
+      setHuman(payload);
+    }
+    // here setVisible(false);
```
Comment (Karen-Avdalyan) [inline src/App.js:32]: delete the comment

File: src/App.js [hunk 5]
```diff
@@ -1,24 +1,88 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
+      <Input
+        type='text'
```
Comment (Karen-Avdalyan) [inline src/App.js:46]: you can put default value to this prop, to not write it every time

File: src/App.js [hunk 6]
```diff
@@ -1,24 +1,88 @@
...
+        fieldName='First name'
+        functionality={setFirstName}
```
Comment (Karen-Avdalyan) [inline src/App.js:48]: change the prop name

File: src/Style/styles.css [hunk 7]
```diff
@@ -0,0 +1,49 @@
+body {
+  background-color: rgb(0, 0, 0);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+input[type=text], [type=number] {
```
Comment (Karen-Avdalyan) [inline src/Style/styles.css:9]: you can to not write here types

File: src/Style/styles.css [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+  font-family: serif;
+  font-size: 20px;
+}
+
+button:hover {
+  background-color: rgb(161, 151, 151);
+  color: black;
+}
+
+div {
```
Comment (Karen-Avdalyan) [inline src/Style/styles.css:35]: I don't think that this is a good idea to give every div the same style. Use class instead.

**Review by Karen-Avdalyan — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 5 above)
Comment (SergeyGhazaryan) [reply src/App.js:46]: I have number type also

**Review by Karen-Avdalyan — APPROVED** [review]

**Pull request merged**